versions/3.1.1.md

```diff
@@ -146,7 +146,8 @@ Data types in the OAS are based on the types supported by the [JSON Schema Speci
 Note that `integer` as a type is also supported and is defined as a JSON number without a fraction or exponent part. 
 Models are defined using the [Schema Object](#schemaObject), which is a superset of JSON Schema Specification Draft 2020-12.
 
-<a name="dataTypeFormat"></a>As defined by the [JSON Schema Validation vocabulary](https://tools.ietf.org/html/draft-bhutton-json-schema-validation-00#section-7.3), data types can have an optional modifier property: `format`.
+<a name="dataTypeFormat"></a>As defined by the [JSON Schema Validation specification](https://tools.ietf.org/html/draft-bhutton-json-schema-validation-00#section-7.3), data types can have an optional modifier property: `format`.  Note that by default, JSON Schema draft 2020-12 validators _do not_ validate `format`, but instead treat it as an annotation.  Support for `format` validation, either in best-effort form with the default `format-annotation` vocabulary, or with the less-commonly-implemented `format-assertion` vocabulary, varies among JSON Schema implementations.
```

Comment:
What are `format-annotation` and `format-assertion`? Please add links.

Reply:
@ralfhandl added!